config/locales/en.yml

```diff
@@ -2189,6 +2216,13 @@ en:
           main_html: <p>We received a request to delete your account in %{school_name}.</p> <p> Please click this %{link_to} to confirm account deletion. The link is valid for 30 minutes only.</p>
           note_html: <p>This action is irreversible, and will permanently remove your access to the school.</p>
         subject: Delete account from %{school_name}
+      email_change_in_user_standing:
+        body:
+          link_to: visit user standing page
+          main_html: <p> Your standing in %{school_name} school has been changed from <strong>%{previous_standing}</strong> to <strong>%{current_standing}</strong>. The following is the reason:</p>
```

Comment:
```suggestion
          main_html: <p> Your standing in %{school_name} school has been changed from <strong>%{previous_standing}</strong> to <strong>%{current_standing}</strong>. Here is the entry:</p>
```

Reply:
Addreseed in https://github.com/pupilfirst/pupilfirst/pull/1468/commits/a519909af207a99fdf3260f5c64a314e2898fd4b